ST_structs.h

```diff
@@ -180,9 +180,10 @@ struct resourcegroup_st {
 		grazingfrq,     /* grazing effect on group at this frequency: <1=prob, >1=# years */
         grazingfreq_startyr;/* start year for grazing frequency*/
   SppIndex *species; /*list of spp belonging to this grp*/
-  RealF min_res_req,  /* input from table */
+  RealF space,  /* input from table */
+        min_res_req,  /* input from table, rescaled if an rgroup is not established */
```

Comment:
change to:
`/* input space from table, rescaled if one or more rgroups is not established */
`